src/crimson/net/Errors.h

```diff
@@ -27,6 +27,7 @@ enum class error {
   read_eof,
   corrupted_message,
   protocol_aborted,
+  shutting_down,
```

Comment:
Might be better if rename to `messenger_down` or `messenger_stopped`.

Reply:
or `network_aborted`.